### PR TITLE
Vulnerable Components Python Function Add

### PR DESCRIPTION
**Pull request by amy0xu**
### Description
Added a function to check if Juice Shop is still vulnerable to zip slip exploit. 

## Checklist
- [x] My code adheres to the coding and style guidelines of the project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.

### CODE REVIEW

File: python/exploits/VulnerableComponents.py [hunk 1]
```diff
@@ -67,6 +75,29 @@ def create_folder(wd_path, folder_name):
...
+    if new_zip_text in response_text:
+        fix_check = False
+
+    return fix_check
+
+
 if __name__ == '__main__':
-    url = r'http://localhost:3000'
-    print(replicate_vulnerable_components(url))
+    url = r'http://localhost:3001'
```
Comment (bnewc) [inline python/exploits/VulnerableComponents.py]: If the port is set to 3001, won't the exploit fail outright?

**Review by bnewc — COMMENTED** [review]
Is the port set incorrectly because there's currently no mitigation for the vulnerability?

File: python/exploits/VulnerableComponents.py (hunk 1 above)
Comment (amy0xu) [reply python/exploits/VulnerableComponents.py]: Changed the port to 3000 now. 

I was testing a local vulnerable Juice Shop on port 3001 and was testing it last.

**Comment by bnewc** [comment]
Looks good!

**Review by bnewc — APPROVED** [review]

**Pull request merged**